src/entities/protocols/uniswap.ts

```diff
@@ -29,9 +29,11 @@ export type FlatFeeOptions = {
 
 // the existing router permit object doesn't include enough data for permit2
 // so we extend swap options with the permit2 permit
+// when safe mode is enabled, the SDK will add extra token sweeps for security
```

Comment:
should we change this to "when tokensToSweep"?

Reply:
oh yeah good catch ty